storage/txcache/txListForSenderAsSortedMapItem_test.go

```diff
@@ -0,0 +1,37 @@
+package txcache
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSenderAsBucketSortedMapItem_ComputeScore(t *testing.T) {
+	list := newListToTest()
+
+	list.AddTx([]byte("a"), createTxWithParams(".", 1, 1000, 200000, 100*oneTrilion))
+	list.AddTx([]byte("b"), createTxWithParams(".", 1, 500, 100000, 100*oneTrilion))
+	list.AddTx([]byte("c"), createTxWithParams(".", 1, 500, 100000, 100*oneTrilion))
+
+	require.Equal(t, uint64(3), list.countTx())
+	require.Equal(t, int64(2000), list.totalBytes.Get())
+	require.Equal(t, int64(400000), list.totalGas.Get())
+	require.Equal(t, int64(40*oneMilion), list.totalFee.Get())
+
+	require.InDelta(t, float64(5.795382396), list.computeRawScore(), delta)
+}
+
+func Test_computeSenderScore(t *testing.T) {
+	require.InDelta(t, float64(0.1789683371), computeSenderScore(senderScoreParams{count: 14000, size: kBToBytes(100000), fee: toMicroERD(300000), gas: 2500000000, minGasPrice: 100}), delta)
```

Comment:
should split in multiple lines

Reply:
Fixed.

Reply:
actually, instead of having 
```GO
score := computeSenderScore(senderScoreParams{count: 14000, size: kBToBytes(100000), fee: toMicroERD(300000), gas: 2500000000, minGasPrice: 100}
```
it is better to have 

```GO
score := computeSenderScore(senderScoreParams{
                count: 14000, 
                size: kBToBytes(100000), 
                fee: toMicroERD(300000), 
                gas: 2500000000, 
                minGasPrice: 100,
}
```